code/modules/mob/living/carbon/carbon.dm

```diff
@@ -276,7 +276,7 @@
 		I = legcuffed
 		type = 2
 	if(I)
-		if(type == 1)
+		if(type == 1 && I.breakout_cooldown)
```

Comment:
Rather than outright not applying the CD, the duration of the CD itself should be variable, ie `var/resist_cd = CLICK_CD_BREAKOUT`.

These should still have SOME cd associated, even if quick (like 1 second)